### PR TITLE
Fix golint issues in test/e2e/lifecycle/

### PR DESCRIPTION
**Pull request by tanjunchen**
**What type of PR is this?**
 
/kind cleanup

**What this PR does / why we need it**:

Fix golint issues in test/e2e/lifecycle/

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/lgtm
for hack
/assign @oomichi

**Review by SataQiu — COMMENTED** [review]
/approve

File: test/e2e/lifecycle/bootstrap/util.go [hunk 1]
```diff
@@ -74,10 +77,12 @@ func addSecretExpiration(s *v1.Secret, expiration string) {
 	s.Data[bootstrapapi.BootstrapTokenExpirationKey] = []byte(expiration)
 }
 
+// TimeStringFromNow returns the time as a string from now
```
Comment (oomichi) [inline test/e2e/lifecycle/bootstrap/util.go]: Could you include an example of the time string to know the format easily?

File: test/e2e/lifecycle/bootstrap/util.go [hunk 2]
```diff
@@ -137,6 +145,7 @@ func WaitForBootstrapTokenSecretToDisappear(c clientset.Interface, tokenID strin
 	})
 }
 
+// WaitForBootstrapTokenSecretNotDisappear waits for bootstrap token secret not to be disappeared
```
Comment (oomichi) [inline test/e2e/lifecycle/bootstrap/util.go]: This function takes time for the specified timeout as success path.
it is nice to describe it at the above.

**Review by oomichi — COMMENTED** [review]
Nice work, thanks.
Just few comments.

File: test/e2e/lifecycle/bootstrap/util.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/lifecycle/bootstrap/util.go]: WaitForBootstrapTokenSecretNotDisappear waits for bootstrap token secret not to be disappeared 
=》
WaitForBootstrapTokenSecretNotDisappear takes time for the specified timeout as success path.
ok?/cc @oomichi

File: test/e2e/lifecycle/bootstrap/util.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/lifecycle/bootstrap/util.go]: TimeStringFromNow(600 * time.Second) 2019-12-03T14:30:40+08:00  ->  2019-12-03T14:30:40+08:00 

// TimeStringFromNow returns the time as a string from now
// 2019-12-03T14:30:40+08:00  TimeStringFromNow(600 * time.Second) 
// and then returns 2019-12-03T14:30:40+08:00 
how about this?Thanks

Comment (oomichi) [reply test/e2e/lifecycle/bootstrap/util.go]: just
```
// TimeStringFromNow returns the time as a string from now
// e.g: 2019-12-03T14:30:40+08:00
```
can be enough :-)

File: test/e2e/lifecycle/bootstrap/util.go (hunk 2 above)
Comment (oomichi) [reply test/e2e/lifecycle/bootstrap/util.go]: How about
```
WaitForBootstrapTokenSecretNotDisappear waits for bootstrap token secret not to be disappeared and takes time for the specified timeout as success path.
```
?

File: test/e2e/lifecycle/bootstrap/util.go (hunk 1 above)
Comment (tanjunchen) [reply test/e2e/lifecycle/bootstrap/util.go]: sure.Thanks

File: test/e2e/lifecycle/bootstrap/util.go (hunk 2 above)
Comment (tanjunchen) [reply test/e2e/lifecycle/bootstrap/util.go]: both.Sure Thanks

**Comment by tanjunchen** [comment]
done.
/cc @oomichi  
thanks

**Review by fejta — COMMENTED** [review]
/approve

**Pull request merged**